### PR TITLE
[platformLibs] mingw platform.posix cleanup

### PR DESCRIPTION
**Pull request by msink**

### CODE REVIEW

File: platformLibs/src/platform/mingw/posix.def [hunk 1]
```diff
@@ -1,13 +1,144 @@
...
+    vadefs.h \
+    wchar.h \
+    wctype.h
+excludedFunctions = __WSAFDIsSet
 noStringConversion = send sendto recv
-linkerOpts = -lWs2_32
+linkerOpts = -lws2_32
 
 ---
+WINSOCK_API_LINKAGE SOCKET WSAAPI accept(SOCKET s,struct sockaddr *addr,int *addrlen);
```
Comment (olonho) [inline platformLibs/src/platform/mingw/posix.def]: Why so?

**Comment by msink** [comment]
Because it's much less ugly way to include `winsock2` to parser but exclude from export.

**Comment by msink** [comment]
Currently windows-specific parts of `winsock2` still leak ko `platform.posix` - `WSA...` functions and structures, `UUID` definition.
After this path they moved to `platform.windows`, as expected.

**Comment by msink** [comment]
POSIX-compatible function definitions were copy-pasted from `winsock2.h` because  POSIX API is stable, really they are the same in `winsock` and `winsock2`, and unlikely will ever changed.

**Pull request closed without merging**